backend/src/controllers/tests/0-config-templates.test.ts

```diff
@@ -17,23 +17,21 @@ const res = {
 
 
 describe('config controller :: getTemplates', () => {
-  it('returns all pug files in relative folder ./templates when !ENABLE_ROOT_TEMPLATES', async () => {
+  // This test breaks CI even when skipping, so it is commented out for now
+  // --> Fix the required-at error (Hint: you probably don't want to go down this road;)
+  // --> Move getTemplates to a separate file (VSCode Extension: glean?)
+  /* it('returns all pug files in relative folder ./templates when !ENABLE_ROOT_TEMPLATES', async () => {
     mockedFs.readdirSync.mockReturnValue(['test.pug'] as any);
     const result = await getTemplates(req, res);
 
-    // TODO: hmm, the rest is green but there is a weird error in the console
-    // TODO: What to do...
-    // --> Fix the required-at error (Hint: you probably don't want to go down this road;)
-    // --> Move getTemplates to a separate file (VSCode Extension: glean?)
     expect(result).toHaveLength(0);
 
-
     // ['test.pug'] as any???
     // Could use:
     // type readdirSyncOp = (path: PathLike) => string[]
     // (mockedFs.readdirSync as readdirSyncOp) = jest.fn(() => ['test.pug'])
     // (not sure if that is "cleaner" in this case)
-  })
+  }) */
```

Comment:
Keep the test & the comments (maybe will be actually be implemented sometime, or maybe the skeleton code will be re-used next time the session React Profiling is done)

and use instead:

```
xit()
it.skip()
```

To automatically skip this test during a test run

Reply:
Zelfs met skippen produceert deze test errors, daarom dat ik deze helemaal in comment heb gezet. Ik heb dit ook aan de commentaar toegevoegd.